### DISCUSSION TITLE
Tried wikki and all searching yet failed to hide this bulb.

### DISCUSSION THREAD

**Discussion by semi710** [nvimdev/lspsaga.nvim#1375] [Q&A]
![image](https://github.com/nvimdev/lspsaga.nvim/assets/60490474/dc3097f1-06dd-4047-93e4-bdefeef6092a)
i want it to come when i am on the line not always.

**Comment by iidexic** [comment]
You can get rid of the lightbulb entirely by adding this to your config options:
`      
ui = {
        code_action = '',
      },
`

This really just makes it invisible.  to actually fully disable, you would have to run
` require('lspsaga').config.lightbulb.enabled = false`
To make it show only on specific lines would maybe take a couple of autocommands and digging into available data/functions vim has by default for quickfix as well as digging into what functions lspsaga provides
